class-archiveless.php

```diff
@@ -81,15 +81,16 @@ public function is_block_editor() {
 	public function setup() {
 		add_action( 'init', array( $this, 'register_post_status' ) );
 		add_action( 'init', array( $this, 'register_post_meta' ) );
+		add_action( 'init', function() {
```

Comment:
Instead of `init:99`, I'd advise we use `wp_loaded`. That _should_ guarantee we catch every post type, whereas `init:99` does not.

I'd also recommend this hook into a method vs an anonymous function. In addition to matching the rest of the hooks here, anonymous functions are extremely difficult to unhook, should a site ever need to do that. While there are plenty of problems with the Singleton pattern, I would offer that this is the biggest "plus" to using it in a WordPress plugin: any other plugin or theme can easily reference `[ Archiveless::instance(), 'rest_prepare_post_types' ]`.

Reply:
Makes sense. Modifications made accordingly.